docs/content/components/label.mdx

```diff
@@ -56,9 +57,7 @@ import { Label } from '@marigold/components';
 ### Label disabled
 
 ```tsx
-<Label variant="disabled" htmlFor="labelId">
-  I am a disabled Label!
-</Label>
+<Label htmlFor="labelId">I am a disabled Label!</Label>
```

Comment:
remove this and some other parts in this PR because Labels are not disabled (e.g. Label/Input/Field)